### PR TITLE
Update User-Agent Client Hints docs

### PR DESCRIPTION
**Pull request by bershanskiy**
#### Summary

Refactor User-Agent client hints docs more.

#### Motivation

Improve prose, add more inter-links to improve discoverability.

#### Supporting details

N/A

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

CC @hamishwillee

### CODE REVIEW

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 1]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
 A {{jsxref("Promise")}} that resolves to an object containing some or all of the following values (based on the hints requested):
 
 - `brands`
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.brands")}}.
+  - : Returns an array of objects containing `brand` and `version` specifying the browser brand and its version (the same information as provided by {{domxref("NavigatorUAData.brands")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `mobile`
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.mobile")}}.
+  - : Returns `true` if the user-agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:49]:
```suggestion
  - : Returns `true` if the user agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 2]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
+  - : Returns an array of objects containing `brand` and `version` specifying the browser brand and its version (the same information as provided by {{domxref("NavigatorUAData.brands")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `mobile`
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.mobile")}}.
+  - : Returns `true` if the user agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Mobile")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `platform`
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.platform")}}.
+  - : Returns a string describing the platform the user-agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:52]:
```suggestion
  - : Returns a string describing the platform the user agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 3]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.mobile")}}.
+  - : Returns `true` if the user agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Mobile")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `platform`
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.platform")}}.
+  - : Returns a string describing the platform the user agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `architecture`
   - : A string containing the platform architecture. For example, `"x86"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:56]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 4]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
-  - : A low-entropy hint provided automatically, for details see {{domxref("NavigatorUAData.platform")}}.
+  - : Returns a string describing the platform the user agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
 - `architecture`
   - : A string containing the platform architecture. For example, `"x86"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `bitness`
-  - : A string containing the architecture bitness. For example, `"64"`.
+  - : A string containing the architecture bitness. For example, `"32"` or `"64"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:59]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 5]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
   - : A string containing the platform architecture. For example, `"x86"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `bitness`
-  - : A string containing the architecture bitness. For example, `"64"`.
+  - : A string containing the architecture bitness. For example, `"32"` or `"64"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `model`
-  - : A string containing the device model. For example, `"Pixel 2XL"` or `""` if device model is not known.
+  - : A string containing the model of mobile device. For example, `"Pixel 2XL"`. If device is not a mobile device or if device model is not known, `model` will be `""`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:62]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 6]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
-  - : A string containing the architecture bitness. For example, `"64"`.
+  - : A string containing the architecture bitness. For example, `"32"` or `"64"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `model`
-  - : A string containing the device model. For example, `"Pixel 2XL"` or `""` if device model is not known.
+  - : A string containing the model of mobile device. For example, `"Pixel 2XL"`. If device is not a mobile device or if device model is not known, `model` will be `""`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `platformVersion`
   - : A string containing the platform version. Platform name itself is always available as low-entropy hint `platform`. For example, `"10.0"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform-Version")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:65]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 7]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
+  - : A string containing the model of mobile device. For example, `"Pixel 2XL"`. If device is not a mobile device or if device model is not known, `model` will be `""`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `platformVersion`
   - : A string containing the platform version. Platform name itself is always available as low-entropy hint `platform`. For example, `"10.0"`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `uaFullVersion` {{deprecated_inline}}
-  - : A string containing the full browser version. For example, `"91.0.4472.124"`.
+  - : A string containing the full browser version. For example, `"103.0.5060.134"`. Deprecated in favor of `fullVersionList`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:68]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 8]
```diff
@@ -43,25 +43,34 @@ getHighEntropyValues(hints)
...
+  - : A string containing the full browser version. For example, `"103.0.5060.134"`. Deprecated in favor of `fullVersionList`.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
 - `fullVersionList`
   - : An array of objects with properties `"brand"` and `"version"` representing the browser name and full version respectively.
-    For example, `{"brand": "Google Chrome", "version": "103.0.5060.134"}, {"brand": "Chromium", "version": "103.0.5060.134"}`.
-    Please note that one object may intentionally contain invalid information to prevent sites from relying on a fixed list of browsers.
+      For example, `{"brand": "Google Chrome", "version": "103.0.5060.134"}, {"brand": "Chromium", "version": "103.0.5060.134"}`.
+      Please note that one object may intentionally contain invalid information to prevent sites from relying on a fixed list of browsers.
+      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version-List")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:73]:
```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version-List")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

File: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md [hunk 9]
```diff
@@ -93,11 +102,13 @@ navigator.userAgentData.getHighEntropyValues(
...
-  - {{HTTPHeader("Sec-CH-UA-Full-Version")}}
-  - {{HTTPHeader("Sec-CH-UA-Mobile")}}
-  - {{HTTPHeader("Sec-CH-UA-Model")}}
-  - {{HTTPHeader("Sec-CH-UA-Platform")}}
-  - {{HTTPHeader("Sec-CH-UA-Platform-Version")}}
+  - [Low-entropy client hints](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) are sent automatically:
+    - {{HTTPHeader("Sec-CH-UA")}}
+    - {{HTTPHeader("Sec-CH-UA-Mobile")}}
+    - {{HTTPHeader("Sec-CH-UA-Platform")}}
+  - Server can request to receive high-entropy client hints on subsequent requests via {{HTTPHeader("Accept-CH")}}:
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md:109]:
```suggestion
  - Servers can request to receive high-entropy client hints on subsequent requests, using the {{HTTPHeader("Accept-CH")}} header:
```

File: files/en-us/web/api/navigatoruadata/mobile/index.md [hunk 10]
```diff
@@ -33,3 +33,7 @@ console.log(navigator.userAgentData.mobile);
 ## Browser compatibility
 
 {{Compat}}
+
+## See also
+
+ - {{HTTPHeader("Sec-CH-UA-Mobile")}} (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) contains the same information.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/mobile/index.md:39]:
```suggestion
 - The {{HTTPHeader("Sec-CH-UA-Mobile")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) contains the same information.
```

File: files/en-us/web/api/navigatoruadata/platform/index.md [hunk 11]
```diff
@@ -34,3 +34,7 @@ console.log(navigator.userAgentData.platform);
 ## Browser compatibility
 
 {{Compat}}
+
+## See also
+
+ - {{HTTPHeader("Sec-CH-UA-Platform")}} (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) contains the same information.
```
Comment (sideshowbarker) [inline files/en-us/web/api/navigatoruadata/platform/index.md:40]:
```suggestion
 - The {{HTTPHeader("Sec-CH-UA-Platform")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) contains the same information.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by hamishwillee** [comment]
I'm late to the party, but this is much better. Thanks @bershanskiy (and @sideshowbarker )

**Pull request merged**